### ISSUE TITLE
[Issue]: Cheats don't work with exFAT SD cards

### ISSUE THREAD

**Issue by Soundtoxin** [emukidid/swiss-gc#783]
### Describe the bug

I have been unable to get .txt format cheats working with Swiss despite following instructions that worked for other people. When I go to a game and press Y to view the cheats, it can't find any. My cheats are stored in a folder called "cheats" at the root of the SD card. I had my friend test cheats the same way on his GameCube and they worked for him. His SD card is FAT32, so I suspect that is the major difference here.

### Steps to reproduce

- Format [micro]SD card as exFAT (or use new SD card that comes this way)
- Load up with Swiss, .txt cheats in "cheats" directory at the root of the card, and a game (e.g. Super Mario Sunshine)
- Insert card into SD2SP2 and insert adapter into the GameCube
- Try to view the cheats for the game by selecting the game file with A and then pressing Y to view cheats

### Expected behavior

Since Swiss itself seems to support exFAT, and my games launch fine from it, I expected the cheats system to work as well.

**Comment by Extrems** [comment]
Cannot reproduce.

**Comment by Soundtoxin** [comment]
I grabbed the cheats collection from here: https://www.gc-forever.com/forums/download/file.php?id=1934

Where from what I can tell they are all using DOS line endings already.

When I run `:set ff?` in vim on a couple of the cheat files it says `fileformat=dos`. I only edited one file and I was deleting rather than adding to it, so line endings shouldn't be different afaik. I did the same check on that file now just in case and it also still said it was dos type. I'll move the cheats to the new spot, but when my friend did his test it sounded like it moves the files for him.
I've done `:e ++ff=dos` and then resaved two files just in case they had some unix line endings in there somehow anyway. (steps taken from the vim wiki https://vim.fandom.com/wiki/File_format )

After testing, the cheats work now! Not sure which change did it. Thanks for the ideas.
